RACK-Ontology/ontology/DO-178C/SECTION_2.csv

```diff
@@ -0,0 +1,82 @@
+description,entityURL,generatedAtTime,identifier,invalidatedAtTime,title,definedIn_identifier,content_identifier,dataInsertedBy_identifier,wasAttributedTo_identifier,wasDerivedFrom_identifier,wasGeneratedBy_identifier,wasImpactedBy_identifier,wasRevisionOf_identifier
+,,,A-1,,Software Planning Process,,A-1-1,,,,,,
+,,,A-1,,Software Planning Process,,A-1-2,,,,,,
+,,,A-1,,Software Planning Process,,A-1-3,,,,,,
+,,,A-1,,Software Planning Process,,A-1-4,,,,,,
+,,,A-1,,Software Planning Process,,A-1-5,,,,,,
+,,,A-1,,Software Planning Process,,A-1-6,,,,,,
+,,,A-1,,Software Planning Process,,A-1-7,,,,,,
+,,,A-2,,Software Development Process,,A-2-1,,,,,,
+,,,A-2,,Software Development Process,,A-2-2,,,,,,
+,,,A-2,,Software Development Process,,A-2-3,,,,,,
+,,,A-2,,Software Development Process,,A-2-4,,,,,,
+,,,A-2,,Software Development Process,,A-2-5,,,,,,
+,,,A-2,,Software Development Process,,A-2-6,,,,,,
+,,,A-2,,Software Development Process,,A-2-7,,,,,,

```

Comment:
Not important for this merge request, but I wonder if we could get away with only ever mentioning "fields" once...